Atmel/Winc15x0/GHIElectronics.TinyCLR.Drivers.Atmel.Winc15x0/Winc15x0Interface.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Runtime.CompilerServices;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using GHIElectronics.TinyCLR.Devices.Network;
+using GHIElectronics.TinyCLR.Devices.Network.Provider;
+
+namespace GHIElectronics.TinyCLR.Drivers.Atmel.Winc15x0 {
+    public class Winc15x0Interface : INetworkControllerProvider {
```

Comment:
Instead of implementing `INetworkControllerProvider` just expose the network controller field as a public property.